.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-	"extends": "klasa/eslint-ts"
+	"extends": "klasa/eslint-ts",
+	"rules": {
```

Comment:
Please don't change lint rules.

Reply:
Those rules are from JavaScript that heavily conflict with their TypeScript counterparts.

- `no-dupe-class-members`: https://github.com/typescript-eslint/typescript-eslint/issues/291
- `no-invalid-this`: https://github.com/typescript-eslint/typescript-eslint/issues/491

Reply:
I agree the best solution would be to modify the rules from `klasa-lint`, but since this was blocking progress in this PR, I kind of rushed them. Do you want me to make a PR for this?

Reply:
You should use specific disables where needed, instead of modifying the lint file as these rules prevent other footguns that will go unnoticed if they are completely disabled globally.

Reply:
There are no footguns, they're rules that are flawed when working with TypeScript (the first forbids one of TypeScript's core features, and the other forbids the ESNext syntax TypeScript rightfully supports since its first release).

Either way, they're fixed in the last test.